Extensions/Firebase/B_firebasetools/D_cloudfirestoretools.ts

```diff
@@ -312,10 +312,10 @@ namespace gdjs {
                 callbackStateVariable.setString('ok');
 
               if (callbackValueVariable)
-                callbackValueVariable.setString(
-                  doc.get(field, { serverTimestamps: 'estimate' }) === undefined
-                    ? 'false'
-                    : 'true'
+                callbackValueVariable.setBoolean(
+                  doc.get(field, { serverTimestamps: 'estimate' }) !==
```

Comment:
Exceptionnally you can either use `==` or better, save the result of `doc.get(field, { serverTimestamps: 'estimate' })` in a temp variable